README.md

```diff
@@ -41,38 +80,221 @@ Checkout [architectures notes](doc/Architecture.md)
+
+#### 5.2.1 Setup DB: Docker
+
+If you already have **Docker** installed in your machine, we prepared a **docker-compose.yml** with all configs ready for your ragnarok server.
+
+Go to */docker* and run:
+```shell
+docker-compose up -d
+```
+
+By default, we ran `/rust-ro/docker/volumes/init.sql`, script that creates a database, user and grant all privileges to the user `ragnarok`. It also copies the `/rust-ro/docker/volumes/pg.sql` volume to your docker instance.
+
+```sql
+-- In case you want to take a look: 
+-- /rust-ro/docker/volumes/init.sql
+
+CREATE USER ragnarok WITH PASSWORD 'ragnarok';
+CREATE DATABASE ragnarok;
+GRANT ALL PRIVILEGES ON DATABASE ragnarok TO ragnarok;
+ALTER DATABASE ragnarok OWNER TO ragnarok;
+```
+
+After your database is up, you will need import the Game SQL (`/rust-ro/pg.sql`) to our new database. For that follow the steps:
+
+* Connect into the Docker `postgres_container` Bash
+* Log-in as user `postgres`
+* Run the `psql` command pointing which the import file to your new database
+
+```shell
+docker exec -it postgres_container bash
+su postgres
+psql ragnarok < pg.sql
+```
+
+With that you're ready to go to the next step. 
+
+#### 5.2.2 Setup DB: Locally
+
+If you have PostgreSQL installed in your machine, you will need to log-in into PSQL and create the user, dabase and give the necessary privilege for it:
+
+```shell
+sudo -u postgres psql
+```
+
+Run the queries below: 
+
+```sql 
+CREATE USER ragnarok WITH PASSWORD 'ragnarok';
+CREATE DATABASE ragnarok;
+GRANT ALL PRIVILEGES ON DATABASE ragnarok TO ragnarok;
+ALTER DATABASE ragnarok OWNER TO ragnarok;
+```
+
+After that, exit pgsql and import our `/rust-ro/db/pg.sql` via cli with:
+
+```shell
+ sudo -u postgres psql ragnarok < db/pg.sql
+```
+
+> If you're using the local version, don't forget to change the `database.port` number to `5433` in your `config.json` file. 
+
+
+### 5.3 Building the Binaries
+
+So far, we have a few executables being compiled together with the project:
+
+- **maps:** TODO description
+- **maps-tool:** TODO description
+- **packets:** TODO description
+- **packets-tool:** TODO description
+- **server:** TODO description
+- **skills-enum-generator:** TODO description
+
+To build, just go on the project root and run: 
+
+```shell
+cargo build
+```
+
+Simple like that! Now you're good to run your servers. 
+
+### 5.4 Running the Binaries
+
+After we have everyting set-up (binaries and database), we should run these binaries to make the game playable:
+
+- server
+- maps
+- packets
+
+
+#### 5.4.1 Running Servers
+
+To run the `server` binary, you will need a `ENV` variable called `DATABASE_PASSWORD` together with your command:
 
+```shell
+DATABASE_PASSWORD=ragnarok cargo run --bin=server
+```
 
+If everything goes right, you should receive something like this output:
+
+```
+exclude pattern sqlx
```

Comment:
from line 229 to237 could be remove as it is not supposed to log those lines (it was for debugging purpose but it remained)